gfx/NeoDays/pngs_tiles_10x10/items/rag_tinder.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": [
+    "rag",
+    "tinder"
```

Comment:
rag and tinder using the same sprite is a bit weird, no?

Reply:
It's how it has been, and I don't have an idea as to what to change the sprite into currently.